hydpy/models/rconc_uh.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+"""Calculate runoff concentration by convolution of the isosceles triangular
+unit hydrograph.
```

Comment:
"isosceles triangular" is not correct anymore.

Suggestion: "Calculate runoff concentration with the unit hydrograph approach."

Reply:
ok

Reply:
The current sentence in the master branch is still the outdated one.  I will fix it when adding "documentation-style" names to the `rconc` model.  However, do you have any memories about further issues that might have gone forgotten?